bundles/org.openhab.binding.irobot/src/main/java/org/openhab/binding/irobot/roomba/RoombaConfiguration.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.irobot.roomba;
+
+/**
+ * Roomba Thing configuration
+ *
+ * @author Pavel Fedin <pavel_fedin@mail.ru> - Initial contribution
+ *
+ */
+public class RoombaConfiguration {
+    public static String FIELD_IPADDRESS = "ipaddress";
+    public static String FIELD_PASSWORD = "password";
```

Comment:
The Configuration class should only contain members that are part of the config.

Reply:
Some API calls refer to names of config fields in string form, i thought it would be very logical and convenient to have them here. But enjoy.